app/search_builders/curation_concerns/search_builder.rb

```diff
@@ -31,22 +31,4 @@ def discovery_permissions
     return ['edit'] if blacklight_params[:works] == 'mine'
     super
   end
-
-  # This is included as part of blacklight search solr params logic
-  def filter_models(solr_parameters)
-    solr_parameters[:fq] ||= []
-    solr_parameters[:fq] << '(' + (work_clauses + collection_clauses).join(' OR ') + ')'
-  end
-
-  def work_clauses
-    return [] if blacklight_params.key?(:f) && Array(blacklight_params[:f][:generic_type_sim]).include?('Collection')
```

Comment:
No specs failed when either was removed (and they generate pretty similar queries), so I'm fine keeping filter_models instead.  I'll open a new PR for that.